tests/unit/resources/test_policies.py

```diff
@@ -141,6 +141,20 @@ def test_get_vms_not_found(self, mock_get):
 
         mock_get.assert_called_once_with('/policies/ABCDE/virtual_machines')
 
+    @mock.patch.object(Connection, "post")
+    @mock.patch.object(Connection, "get")
+    def test_create_policy(self, mock_post, mock_get):
+        resource_data = [{'name': 'test'}]
```

Comment:
your resource data should have an id in it if you are going to use get_by_id

Reply:
Added the id parameter in the resource data as per the review comment